src/App.js

```diff
@@ -26,6 +27,7 @@ function App() {
             <Dashboard />
           </Route>
           <Route exact path="/loans" component={Loans} />
+          <Route exact path="/ships" component={Ships} />
```

Comment:
Personal Preference:
You could add an array at the top and loop through it here. So you safe yourself some code writing. Like:
```suggestion
function App() {
...
  const routes = [
    { path: '/', Component: Dashbaord, exact: true },
    { path: '/loans', Component: Loans },
    { path: '/ships', Component: Ships },
  ]
  return (
      <>
      <h1>Space Traders</h1>
      <Switch>
        {routes.map(({ Component, ...routeProps }) => (
          <Route key={routeProps.path} {...routeProps}>
            <Component />
          </Route>
        ))}
      </Switch>
      </>
  )
}
          <Route exact path="/ships" component={Ships} />
```